Specs/add_czesc.rb

```diff
@@ -0,0 +1,7 @@
+def add_czesc(chapter5)
```

Comment:
```suggestion
def add_czesc(sentence)
```

I'd use generic name for the variable.

Look at the description of the test for this method: "adds czesc at the beggining of the sencence". It is not about adding something to chapter 5, but to generic sentence.

Reply:
alternatively you could use `name` instead of `sentence` (that would be even better imo)